### PR TITLE
If file has not changed but Etag has, this change must be reflected n the local db

### PR DESCRIPTION
**Pull request by allexzander**
Signed-off-by: alex-z <blackslayer4@gmail.com>

### CODE REVIEW

File: src/libsync/propagatedownload.cpp [hunk 1]
```diff
@@ -465,6 +465,15 @@ void PropagateDownloadFile::start()
         return;
     }
 
+    SyncJournalFileRecord record;
+    const auto etagChangedButFileHasNotChanged = propagator()->_journal->getFileRecord(_item->_file, &record) && record.isValid()
```
Comment (claucambra) [inline src/libsync/propagatedownload.cpp:469]: I would put each condition in a separate line for readability

**Review by claucambra — APPROVED** [review]
Just a small comment, looks good

**Review by mgallien — CHANGES_REQUESTED** [review]
this should come with changes in the automated tests
otherwise this could break or be broken in the future

**Comment by allexzander** [comment]
> this should come with changes in the automated tests otherwise this could break or be broken in the future
Agree, and I should have used a completely another way.

**Comment by allexzander** [comment]
Continue working on it in a new PR https://github.com/nextcloud/desktop/pull/5293

**Pull request closed without merging**